e2e/testdata/snapshot_restore/snapshotclass.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: snapshot.storage.k8s.io/v1
```

Comment:
Should this be part of e2e/manifests/common?

Reply:
I didn't add the volumesnapshotclass deployment by default in the cluster as the snapshot feature is just partially supported by Topolvm. Although, I can add it to the regular deployment if you think. Thoughts?

Reply:
Sure, added the volumesnapshot deployment to manifests/common